src/improvements/README.md

```diff
@@ -57,7 +63,7 @@ SIMULATE_WITHOUT_LEDGER=1 just --dotenv-path $(pwd)/.env --justfile ../../../nes
 SIMULATE_WITHOUT_LEDGER=1 just --dotenv-path $(pwd)/.env --justfile ../../../single.just simulate
 ```
 
-4. Fill out the `README.md` and `VALIDATION.md` files.
+5. Fill out the `README.md` and `VALIDATION.md` files.
```

Comment:
TODO: Blaine add in a short section that explains adding state overrides to your tasks. This now happens for every tasks so it'll be useful to add .

also add in these small fixes:
Make sure to create VALIDATION.md file automatically via just new task
Give better error if someone tries to run a single task as nested.
Make README.md file that’ created from just new task be better. 
Remove timing section from README and any other sections that add extra steps.
Remove extra spacing from printed markdown title: 7-L60C59)  (ProxyAdmin) - Chain ID: 130